src/components/eip3643/user/TransferToken.tsx

```diff
@@ -59,6 +66,60 @@ export default function TransferToken({
     form.values.toAddress,
   );
 
+  const updateMostUsedAddresses = (value: string) => {
+    const items = localStorage.getItem("mostUsedAddresses");
```

Comment:
Maybe we can simplify the dealing with LS with some ready-made hooks like
https://usehooks-ts.com/react-hook/use-local-storage
or 
https://github.com/streamich/react-use/blob/master/docs/useLocalStorage.md